development/cli/getting_started.rst

```diff
@@ -0,0 +1,67 @@
+===============
+Getting started
+===============
+
+The command line interface (CLI) is a useful utility for phpBB administrators who have access to a shell on the server they run phpBB on, as well as for extension developers. It provides CLI commands for managing config values, extensions, running database migration, purging the cache, and more.
+
+Most operating systems ship with a built in command line application. For Mac OS and Ubuntu it is called "Terminal" and for Windows it is called "Command Prompt". Third party software such as PuTTy and iTerm can also be used.
+
+To use phpBB's CLI on a web server, you will need SSH access to your web server. You can find out from your web hosting company if they offer SSH access and how to log into your web server from the CLI.
+
+Once you have accessed your web server via SSH, or if you are in a localhost development environment, you simply need to navigate to your phpBB forum directory via the ``cd`` command:
+
+.. code-block:: console
+
+    $ cd /path/to/phpBB
```

Comment:
Relative path, not absolute, this will confuse people on shared hosting I think.